README.md

```diff
@@ -64,16 +64,6 @@ Install Node.js 16 first (or if you use [nvm](https://github.com/nvm-sh/nvm), sw
 
 ```bash
 npm install --global --production windows-build-tools
-npm config set msvs_version 2017
-```
-</details>
-
-<details><summary>Extra dependencies for GNU/Linux</summary>
```

Comment:
How about we keep all of this, but clarify it's only needed for development?

Reply:
Agreed, have added them back in

Reply:
Only the `npm config set msvs_version 2017` is no longer necessary, thus removing that while at it.